src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -202,92 +205,99 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
 
             var currentLoader = _loader;
 
-            _joinableTaskFactory.Value.RunAsync(async () =>
-            {
-                await TaskScheduler.Default;
+            _joinableTaskFactory.Value.RunAsync(
+                    () => RepopulatePackageList(selectedPackageItem, currentLoader, loadCts)
+                )
+                .FileAndForget(TelemetryUtility.CreateFileAndForgetEventName(
+                    nameof(InfiniteScrollList),
+                    nameof(RepopulatePackageList)
+                ));
+        }
 
-                var addedLoadingIndicator = false;
+        private async Task RepopulatePackageList(PackageItemListViewModel selectedPackageItem, IPackageItemLoader currentLoader, CancellationTokenSource loadCts)
```

Comment:
Nothing change in the method here right?
It's only for reporting purposes correct? I want to make sure I didn't miss something.

Reply:
That's right, was an "extract method".